set.mm

```diff
@@ -461998,19 +462001,18 @@ Class abstractions (a.k.a. class builders)
     CAHUAUFAIACJKLMUAUEACBEZFZUCUAUECUDEZUGFUHCUDBNUAUIAUGUAUIAIACOKPQACBRST $.
 
   $( Condition for a class union to be a subset.  (Contributed by Thierry
-     Arnoux, 21-Jun-2020.) $)
+     Arnoux, 21-Jun-2020.)  Use ~ sspwuni instead.
+     (Proof modification is discouraged.)  (New usage is discouraged.) $)
   pwuniss $p |- ( A C_ ~P B -> U. A C_ B ) $=
     ( cpw wss cuni uniss unipw sseqtrdi ) ABCZDAEIEBAIFBGH $.
```

Comment:
Yes, feel free to delete, thank you!

Reply:
Done in https://github.com/metamath/set.mm/pull/3909/commits/a10ec2ed2dde9e2a0ff863dc2dfc0828cef2b0e7